### PR TITLE
implementation of getPadHistory

### PR DESCRIPTION
**Pull request by athineouc**
getPadHistory function take with post request a pad id and return the whole history of this pad

### CODE REVIEW

File: src/Controller/controllers.go [hunk 1]
```diff
@@ -105,21 +130,81 @@ func (c Controller) LoadFile(w http.ResponseWriter,
...
+	//close th db
+	defer db.Close()
+
+	//query to db to take the history of pad
+	sqlStatement := `SELECT * FROM historyFiles WHERE id=?`
+	rows, err := db.Query(sqlStatement, padRequest.Id)
+	//iterate the results from query
+	for rows.Next() {
+		//read the values per row
+		err = rows.Scan(&ip, &id, &time, &state)
```
Comment (ManosKokos) [inline src/Controller/controllers.go:186]: Time in db is TIMESTAMP and here it is string, sure you want to do it this way???

File: src/Controller/controllers.go [hunk 2]
```diff
@@ -105,21 +130,81 @@ func (c Controller) LoadFile(w http.ResponseWriter,
...
+		if err != nil {
+			errorFlag = true
+		}
+		//insert them to the slice
+		historyToInsert := PadHistory.PadHistory{ip, state, time}
+		history = append([]PadHistory.PadHistory{historyToInsert}, history...)
```
Comment (ManosKokos) [inline src/Controller/controllers.go:192]: Appending to a new slice of PadHistory containing only one item the whole already existing slice, also doing a
unnecessary insertion/allocation of new variable.
Please change the 191+192 to: `history  = append(history , PadHistory.PadHistory{ip. state,time})`
Or again you can get rid of ip,state,time,id and have one var of pad history and have one
var h PadHistory.PadHistory ,scan directly to it and append
...
err = rows.Scan(&h.ip, &h.id, &h.time, &h.state)
...
history = append(history, h)
....

**Review by ManosKokos — COMMENTED** [review]
Please check comments

File: src/Controller/controllers.go (hunk 1 above)
Comment (parriiss) [reply src/Controller/controllers.go:186]: I think it will be ok.We just want to show it in the page so string i think is fine

**Pull request merged**